pytest_django/plugin.py

```diff
@@ -327,6 +329,86 @@ def restore():
         request.addfinalizer(restore)
 
 
+@pytest.fixture(autouse=True, scope='session')
+def _fail_for_invalid_template_variable(request):
+    """Fixture that fails for invalid variables in templates.
+
+    This fixture will fail each test that uses django template rendering
+    should at template contain an invalid template variable.
+
+    It does not raise an exception, but fails, as the stack trace doesn't
+    offer any helpful information to debug.
```

Comment:
This should mention that it fails with a message, possibly mentioning the template, but at least the variable.
My first impression from reading was that it would "just fail".